File: CS/Form1.cs
using System;
using System.Windows.Forms;

namespace RepOverrideRtfFormatting {
    public partial class Form1 : Form {
        private XtraReport1 report = new XtraReport1();
        
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            report.OverrideRtfFormatting = checkBox1.Checked;
            report.CreateDocument();
            report.ShowPreviewDialog();
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace RepOverrideRtfFormatting

    Public Partial Class Form1
        Inherits Form

        Private report As XtraReport1 = New XtraReport1()

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            report.OverrideRtfFormatting = checkBox1.Checked
            report.CreateDocument()
            report.ShowPreviewDialog()
        End Sub
    End Class
End Namespace
